plugins/modules/panos_active_in_ha.py

```diff
@@ -29,7 +29,7 @@
     - A simple boolean check, verifies if a node is an active (B(true)) or passive (B(false)) node in an HA pair.
     - If node does not belong to an HA pair or the pair is no configured correctly the module will fail.
 author: "Łukasz Pawlęga (@fosix)"
-version_added: '2.18.0'
+version_added: '2.20.0'
```

Comment:
Hmm, I wasn't sure about that one and tried to figure it out together with @horiagunica. We didn't understand the point of this versioning at all but since it was there, we thought it makes sense to bump it. I'll double check with Adam to be 100% sure. Anyway, thanks!

Reply:
I talked to Adam, he says that it may be worth bumping only when significant breaking changes are introduced. I switch it back to `2.18.0`.